### PR TITLE
chore: Add warning for upcoming removal of static masters

### LINKED ISSUE #6887: Add log warning for static master removal to all jx commands run with static master cluster
With static master removal coming soon, we should start warning users that it's coming.

### PR DESCRIPTION
**Pull request by abayer**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description

This message should show up in the following situations:
* Running locally, connected to a cluster using static master.
* Running locally, calling `jx create cluster` without `--skip-installation` and without flags for installing with Tekton.
* Running locally, calling `jx install` without flags for installing with Tekton.

It should never show up in pipelines, in controllers, or in any case where you're connected to a Tekton cluster.

#### Which issue this PR fixes

fixes #6887

### CODE REVIEW

**Comment by abayer** [comment]
This is aimed to merge on March 11, once there's a blog post on the removal of static masters for this to link to.

File: pkg/cmd/cmd.go [hunk 1]
```diff
@@ -348,6 +349,62 @@ func setLoggingLevel(cmd *cobra.Command, args []string) {
...
+
+		teamSettings := &env.Spec.TeamSettings
+		shouldWarn = !teamSettings.IsJenkinsXPipelines()
+	}
+
+	if shouldWarn {
+		cmd.Println("WARNING")
+		cmd.Println("")
+		cmd.Println("Traditional Jenkins masters are being deprecated and removed from all new Jenkins X releases starting in April, 2020.")
+		cmd.Println("We strongly recommend not using this configuration and instead using the default Tekton-based configuration instead for all Jenkins X activity going forward.")
```
Comment (MarckK) [inline pkg/cmd/cmd.go]: Log message has 'instead' twice. Possible replacement: "We strongly recommend using the default Tekton-based configuration for all Jenkins X activity going forward. We do not recommend using Jenkins masters."

Comment (abayer) [reply pkg/cmd/cmd.go]: How about `We strongly recommend using the default Tekton-based configuration instead for all Jenkins X activity going forward. We do not recommend using the Jenkins X workflow based around Jenkins masters.`? "We do not recommend using Jenkins masters" is a little too potentially misleading. =)

Comment (ethangj) [reply pkg/cmd/cmd.go]: I'm good with `We strongly recommend using the default Tekton-based configuration instead for all Jenkins X activity going forward. We do not recommend using the Jenkins X workflow based around Jenkins masters.`

**Pull request merged**